### PR TITLE
MHR Information Tombstone changes

### PR DESCRIPTION
**Pull request by owillborn**
*Issue #:* /bcgov/entity#15943

*Description of changes:*
- Removed Expiry Date from tombstone in MHR Transfer flows
- Changed Registration Date and Time to Registration Status
- Verified old behavior is retained in PPR
image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/tombstone/TombstoneDischarge.vue [hunk 1]
```diff
@@ -75,6 +93,10 @@ export default defineComponent({
         }
         return 'No Expiry'
       }),
+      statusType: computed((): string => {
+        if (getMhrInformation.value) return getMhrInformation.value.statusType
+        return 'N/A'
+      }),
```
Comment (cameron-eyds) [inline ppr-ui/src/components/tombstone/TombstoneDischarge.vue]: Optional chaining is your friend here :)
`return getMhrInformation.value?.statusType || 'N/A'`

**Review by cameron-eyds — APPROVED** [review]
1 small comment, looks good to me!

**Review by doug-lovett — APPROVED** [review]
Looks good to me.

File: ppr-ui/src/components/tombstone/TombstoneDischarge.vue (hunk 1 above)
Comment (owillborn) [reply ppr-ui/src/components/tombstone/TombstoneDischarge.vue]: missed opportunity, thanks!

**Review by dimak1 — APPROVED** [review]

**Pull request merged**